data/json/item_groups.json

```diff
@@ -6560,6 +6560,7 @@
       ["bio_painkiller", 10],
       ["bio_alarm", 10],
       ["bio_ears", 10],
+	  ["bio_earplugs", 10],
```

Comment:
You're using tabs here. That's definitely not good, please replace them here and below by decent amount of whitespaces.